### PR TITLE
Codespaces `write-all` permission syntax is incorrect

### PR DESCRIPTION
**Pull request by markphelps**
Codespaces `write-all` permission syntax is incorrect.

The current docs show invalid json

### What's being changed:

```
"permissions": {
  "write-all"
 }
```

is invalid JSON.

It should be:

```
"permissions": "write-all"
```

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by reybard — DISMISSED** [review]

File: content/codespaces/managing-your-codespaces/managing-repository-access-for-your-codespaces.md [hunk 1]
```diff
@@ -95,9 +95,7 @@ To create codespaces with custom permissions defined, you must use one of the fo
       "codespaces": {
         "repositories": {
           "my_org/my_repo": {
-            "permissions": {
-              "write-all"
-            }
+            "permissions": "write-all"
```
Comment (jkeech) [inline content/codespaces/managing-your-codespaces/managing-repository-access-for-your-codespaces.md:98]: Should we also change line 90 above to say 

```
To set all permissions for a given repository, use `"permissions": "read-all"` or `"permissions": "write-all"` in the repository object
```

**Review by jkeech — DISMISSED** [review]
Thanks! Left one suggestion about a copy change directly above this that might be nice to tweak as well. ✨

**Review by jkeech — APPROVED** [review]

**Comment by ramyaparimi** [comment]
@markphelps 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by truong1991 — COMMENTED** [review]
> #17657

**Review by hubwriter — APPROVED** [review]
@markphelps - Many thanks for making this correction. 🎖️

**Pull request merged**